handlers.go

```diff
@@ -179,6 +217,14 @@ func (b *BouncerHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	product = strings.TrimSpace(strings.ToLower(product))
 	os = strings.TrimSpace(strings.ToLower(os))
 
+	// HACKS
+	// If the user is coming from windows xp, send a sha1
+	// signed product.
+	// HACKS
+	if isWindowsXPUserAgent(req.UserAgent()) {
```

Comment:
Should we only do this when a windows build is being requested? This would mean that 43.0.1 is the only version it's possible to download from windows xp even if you're purposefully downloading for a different platform. I can't imagine that happens much, but...